source/core/annotations/ut_annotation_manager.pkb

```diff
@@ -267,10 +267,11 @@ create or replace package body ut_annotation_manager as
       l_sql_text    ora_name_list_t := a_sql_text;
```

Comment:
Is there a need for that variable anymore its passed as attribute anyway.

Reply:
Good spot - fixed